app/controllers/businessmatching/updateservice/ChangeBusinessTypesController.scala

```diff
@@ -43,7 +43,7 @@ class ChangeBusinessTypesController @Inject()(
                                           val router: Router[ChangeBusinessType]
                                         ) extends BaseController with RepeatingSection {
 
-  def get = Authorised.async {
+  def get(edit: Boolean = false) = Authorised.async {
```

Comment:
I don't believe we need an edit flag on this page as it's not something we can 'edit' as such, it's never pre-populated nor navigated to from a summary page.

Reply:
Edit functionality removed